google-cloud-bigquery/src/main/java/com/google/cloud/bigquery/CsvOptions.java

```diff
@@ -192,6 +205,14 @@ public Long getSkipLeadingRows() {
     return skipLeadingRows;
   }
 
+  /**
+   * Returns whether BigQuery should allow ascii control characters in a CSV file. By default ascii
```

Comment:
nittiest of nits:  ASCII vs ascii for prose.

Reply:
Done